docs/configuration-reference.md

```diff
@@ -2821,3 +2824,76 @@ Full reference of options:
   # Timeout for connecting to memcached.
   [timeout: <duration> | default = "1s"]
 ```
+
+### postgres_exporter_config
+
+The `postgres_exporter_config` block configures the `postgres_exporter`
+integration, which is an embedded version of
+[`postgres_exporter`](https://github.com/wrouesnel/postgres_exporter). This
+allows for the collection of metrics from Postgres servers.
+
+Full reference of options:
+
+```yaml
+  # Enables the postgres_exporter integration, allowing the Agent to automatically
+  # collect system metrics from the configured postgres server address
+  [enabled: <boolean> | default = false]
+
+  # Automatically collect metrics from this integration. If disabled,
+  # the postgres_exporter integration will be run but not scraped and thus not
+  # remote-written. Metrics for the integration will be exposed at
+  # /integrations/postgres_exporter/metrics and can be scraped by an external
+  # process.
+  [scrape_integration: <boolean> | default = <integrations_config.scrape_integrations>]
+
+  # How often should the metrics be collected? Defaults to
+  # prometheus.global.scrape_interval.
+  [scrape_interval: <duration> | default = <global_config.scrape_interval>]
+
+  # The timeout before considering the scrape a failure. Defaults to
+  # prometheus.global.scrape_timeout.
+  [scrape_timeout: <duration> | default = <global_config.scrape_timeout>]
+
+  # Allows for relabeling labels on the target.
+  relabel_configs:
+    [- <relabel_config> ... ]
+
+  # Relabel metrics coming from the integration, allowing to drop series
+  # from the integration that you don't care about.
+  metric_relabel_configs:
+    [ - <relabel_config> ... ]
+
+  # Monitor the exporter itself and include those metrics in the results.
+  [include_exporter_metrics: <bool> | default = false]
+
+  #
+  # Exporter-specific configuration options
+  #
+
+  # Data Source Names specifies the Postgres server(s) to connect to. This is
+  # REQUIRED but may also be specified by the POSTGRES_EXPORTER_DATA_SOURCE_NAME
+  # environment variable. If neither are set, the integration will fail to
+  # start.
+  #
+  # The format of this is specified here: https://pkg.go.dev/github.com/lib/pq#ParseURL
+  #
+  # A working example value for a server with a password is:
+  # "postgresql://username:passwword@localhost:5432/database?sslmode=disable"
+  #
+  # Multiple DSNs may be provided here, allowing for scraping from multiple
+  # servers.
```

Comment:
optional: Do you think it might be worth mentioning the format in which env var `POSTGRES_EXPORTER_DATA_SOURCE_NAME ` expects multiple DSNs? i.e. specifying it expects a comma separated list (rather than e.g. setting the env multiple times which I feel like I've seen before)

Reply:
Good idea, thanks!